### ISSUE TITLE
jsconfig.json is not used by default like tsconfig.json is

### ISSUE THREAD

**Issue by JoelEinbinder** [Microsoft/TypeScript#18235]
**TypeScript Version:**  2.4.2

I have a `jsconfig.json` file in my current directory and run:
```bash
tsc -p .
```

**Expected behavior:**
It uses the `jsconfig.json` as the config file.

**Actual behavior:**
```
error TS5057: Cannot find a tsconfig.json file at the specified directory: '.'.
```

This is inconsistent with the behavior in VSCode where either `tsconfig.json` or `jsconfig.json` are used.

**Comment by DanielRosenwasser** [comment]
I don't think we have an intention of using `jsconfig.json` as a mechanism for driving TypeScript projects. `jsconfig.json` was always more for accomodating JavaScript users so as not to push them into TypeScript if they don't want, but I would much rather we keep a consistent and explicit way to create TypeScript projects.

**Comment by JoelEinbinder** [comment]
It is weird that https://github.com/Microsoft/TypeScript/blob/d7b3b6c77d45adc4542c882a3f19d2f0fdb3e76c/src/server/editorServices.ts#L971 is inconsistent with the default behavior though. Not a big deal either way, but not having `jsconfig.json` work as a default pushes me to always use `tsconfig.json`. This removes the nice ability to see at a glance whether project is in TypeScript or just using `tsc` to lint JavaScript.

**Comment by mhegazy** [comment]
Automatically closing this issue for housekeeping purposes. The issue labels indicate that it is unactionable at the moment or has already been addressed.
